tools/build_pak_files.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import glob
+import os
+import subprocess
+import sys
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--unreal_binaries_dir", required=True)
+    parser.add_argument("--unreal_project_dir", required=True)
```

Comment:
unreal_project_dir should be optional and handled in the same way as in generate_config.py

Reply:
Donw

Reply:
Update: I've updated my belief here, I now think that it is ok to hard-code the unreal_project_dir to `SpearSim` because I think we'll always want to build specific content for specific unreal projects.

Reply:
I agree. Updated code to only handle `SpearSim`.